chapters/en/module4.md

```diff
@@ -160,38 +150,61 @@ You can hard-reset your work in both cases, but one of the options could cause p
 </choice>
 </exercise>
 
-<exercise id='7' title='Revert your changes' type='slides, video'>
+<exercise id='7' title='Revert changes made to your Git project' type='slides, video'>
 <slides source='module4/module4_04' shot='0' start='3:42' end='4:35'> </slides>
 </exercise>
 
-<exercise id='8' title='Test Your Knowledge'>
+<exercise id='8' title='✍️ Practice: Subtraction through addition (of a new commit)'>
 
 Which of the following statements are TRUE about `git revert`?
 
-
-<choice>
-<opt text='This command creates a new commit that undoes the changes from a previous commit.' >
+<choice id="1">
+<opt text='This command creates a new commit that undoes the changes from previous commits.' >
 </opt>
+That's right, but there is also another correct option!
 <opt text='This command undoes the changes in your project up to the commit id that you have specified.'>
+That's right, but there is also another correct option!
 </opt>
-<opt text='This command makes the history of the project shorter (deletes commits)'>
+<opt text='This command makes the history of the project shorter by deleting commits'>
+Resetting deletes commits, but reverting does not.
 </opt>
 <opt text='This command is used to undo changes in your working directory that have not been comitted yet.'>
+We cannot use revert for this task.
 </opt>
-<opt text='A and B are correct' correct='true'>
+<opt text='The first and second options are both correct' correct='true'>
 </opt>
-<opt text='A and C are correct'>
+<opt text='The first and third options are both correct'>
+Is the third options really correct?
 </opt>
 </choice>
 
+You are writing a book using version control. In your local repository, you have a text file for each chapter. Today, after reading your work again you decided to move some of the text you have written for `chapter3.md` to `chapter1.md`. How would you save this change using Git?
```

Comment:
I find this exercise a bit confusing due to the long pieces of text. What is your opinion about it?

Reply:
I made a minor clarification here, but overall I think it is OK to have one exercise with a little longer text as long as it is clear and the other exercises are shorter.